docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,202 @@
+
+RAG's ability to take advantage of specialization through model orchestration gives RAG an advantage in solving multiple complex problems:
+
+* reducing the overall memory footprint; small models well integrated in the pipeline can compete with gigantic frontier models
+* ensuring that a model can run locally and process private organizational data without liability
+* limiting the amount of work needed to deploy the model in production; many requirements (source verifiability, better grounding, better understanding of original sources) can be addressed at the pretraining level
+
+At PleIAs, we aim to move even further in this direction. While several intermediaries focus on fitting internal data better for LLM processing, we work to bring models closer to the type of content actually used in production. Our current generation of models is trained on a large corpus of non-HTML documents (especially PDF) in a wide variety of formats and intentionally featuring many examples of digitization artifacts (OCR noise, bad segmentation) - a model can only get good at correcting things it has seen.
+
+As with any integrated search and document processing system, a critical aspect of any RAG application is redundancy and fallbacks. Everything fails at some point - the unexpected query, the wrong document, the bad token. While pre-processing the RAG dataset achieves a lot, we believe that some of the redundancy and fallback burden should fall on the model itself, to ensure that it's sufficiently resilient to the content people actually use.
+
+Another overlooked factor for propelling a new wave of specialized or retrained models is that RAG datasets are growing. Even for applications with a well-targeted audience, there are strong incentives to expand the range of indexed context - to cover more use cases. The improved quality and efficiency of search retrieval reverses means that content that is less related (for instance generalist newspaper articles or even some parts of common crawl) is increasingly usable.
+
+A very large RAG dataset, properly documented and augmented with synthetic data, looks a lot like pretraining data. Both large scale RAG and the recent focus of pre-training on “quality data” invites us to think about “data orchestration”. Synthetic data is a hot topic in pre-training right now, not just as a source of new data, but as a powerful tool to process, clean and expand on existing resources. I’m expecting the same process to reverberate on the production side. It’s even likely that, in the near future, because data transactions have the potential to cut costs on model transactions, most RAG calls will return pre-generated texts.
+
```

Comment:
"well-targeted audience"
do you mean:
a) an audience whose interests, behavior etc are well known?
b) an audience that is very particular / specific?

Reply:
Ah i meant b. So maybe something like "specialized applications with a limited audience"